### PR TITLE
Add a default timezone arg to jsonld import, re yale#9708

### PR DESCRIPTION
**Pull request by ryan86**
Add timezone flag to jsonld import to indicate timezone for data that may not include a timezone by default, re #9708

### CODE REVIEW

File: arches/app/utils/data_management/resources/formats/rdffile.py [hunk 1]
```diff
@@ -650,6 +650,13 @@ def find_matching_branch(self, k, v, tree_node, result, tile=None, indent=0):
                         else:
                             self.printline(f"Could not validate {values} as a {o['datatype']}", indent + 1)
                     else:
+                        if o["datatype"].datatype_name == 'date':
+                            try:
```
Comment (apeters) [inline arches/app/utils/data_management/resources/formats/rdffile.py:654]:
```suggestion
                        if self.default_timezone and o["datatype"].datatype_name == 'date':
```

Comment (ryan86) [reply arches/app/utils/data_management/resources/formats/rdffile.py]: Won't this fail if the value can't be parsed on this line

`datetime.datetime.strptime(value, "%Y-%m-%d %H:%M:%S")`

Comment (apeters) [reply arches/app/utils/data_management/resources/formats/rdffile.py:654]: sorry meant my suggestion to be this:
```suggestion
                        if self.default_timezone and o["datatype"].datatype_name == 'date':
                            try:
```

**Review by apeters — APPROVED** [review]
looks good

**Pull request merged**